integration/kotlinx-coroutines-guava/test/ListenableFutureTest.kt

```diff
@@ -316,6 +317,7 @@ class ListenableFutureTest : TestBase() {
     }
 
     @Test
+    @Ignore  // TODO: propagate cancellation before running listeners.
```

Comment:
File and link an issue in the TODO(). If not with this PR, in a followup.

Reply:
Yep, will do.